nemo/collections/common/data/lhotse/nemo_adapters.py

```diff
@@ -0,0 +1,265 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import random
+import re
+import secrets
+import tarfile
+from io import BytesIO
+from pathlib import Path
+from typing import Iterable, List
+
+try:
+    from lhotse.lazy import ImitatesDict
+except ImportError:
```

Comment:
catch both ImportError and ModuleNotFoundError (some python versions raise the latter)